geps/gep-3793/index.md

```diff
@@ -189,10 +194,274 @@ will need resolution before this GEP can graduate.
+much something that we should expect Ana to do often, whether or not default
+Gateways are in play; `gwctl` or something similar SHOULD be able to show her
+which Routes are bound to which Gateways in every case, not just with default
+Gateways.
+
+##### Open Questions
+
+Should the Gateway also add a `condition` explicitly expressing that the Route
+has been claimed by the default Gateway, perhaps with `type: DefaultGateway`?
+This could help tooling like `gwctl` more easily enumerate Routes bound to the
+default Gateway.
+
+#### 2. Controlling which Gateway is the Default
+
+Since Chihiro must be able to control which Gateway is the default, selecting
+the default Gateway must be an active configuration step taken by Chihiro,
+rather than any kind of implicit behavior. To that end, the Gateway resource
+will gain a new field, `spec.isDefault`:
+
+```go
+type GatewaySpec struct {
+    // ... other fields ...
+    IsDefault *bool `json:"isDefault,omitempty"`
+}
+```
+
+If `spec.isDefault` is set to `true`, the Gateway MUST claim Routes that have
+specified no `parentRefs` (subject to the usual Gateway API rules about which
+Routes may be bound to a Gateway), and it MUST update its own `status` with a
+`condition` of type `DefaultGateway` and `status` true to indicate that it is
+the default Gateway, for example:
+
+```yaml
+status:
+  conditions:
+  - type: DefaultGateway
+    status: "True"
+    lastTransitionTime: "2025-10-01T12:00:00Z"
+    message: "Gateway is the default Gateway"
+```
+
+If `spec.isDefault` is not present or is set to `false`, the Gateway MUST NOT
+claim those Routes and MUST NOT set the `DefaultGateway` condition in its
+`status`.
+
+##### Access to the Default Gateway
+
+The rules for which Routes may bind to a Gateway do not change for the default
+Gateway. In particular, if a default Gateway should accept Routes from other
+namespaces, then it MUST include the appropriate `AllowedRoutes` definition,
+and without such an `AllowedRoutes`, a default Gateway MUST accept only Routes
+from its own namespace.
+
+##### Behavior with No Default Gateway
+
+If no Gateway has `spec.isDefault` set to `true`, then the behavior is exactly
+the same as for Gateway API 1.3: all Routes MUST specify `parentRefs` in order
+to function, and no Gateway will claim Routes that do not specify
+`parentRefs`.
+
+##### Deleting the Default Gateway
+
+Deleting the default Gateway MUST behave the same as deleting any other
+Gateway: all Routes that were bound to the default Gateway MUST be unbound,
+and the `Accepted` conditions in the `status` of those Routes SHOULD be
+removed.
+
+##### Multiple Default Gateways
+
+Support for multiple default Gateways in a cluster is not one of the original
+goals of this GEP. However, allowing Chihiro to control which Gateway is the
+default - including being able to switch which Gateway is the default at
+runtime, without requiring downtime - is a goal.
```

Comment:
I think semantically throughout this GEP so far I'm reading the meaning of "Default Gateway" to mean "A single Gateway which routes will apply to by default".

However, I'm wondering if what a "Default Gateway" should more definitively be positioned as is: "a configuration on a Gateway that indicates that it will automatically accept routes within a certain scope by default". Under that definition, I think we would make multiple "Default Gateways" a goal.

:thinking: curious as to your thoughts.